create_topics_request.go

```diff
@@ -20,16 +20,29 @@ func (c *CreateTopicsRequest) setVersion(v int16) {
 	c.Version = v
 }
 
-func NewCreateTopicsRequest(version KafkaVersion, topicDetails map[string]*TopicDetail, timeout time.Duration) *CreateTopicsRequest {
+func NewCreateTopicsRequest(
+	version KafkaVersion,
+	topicDetails map[string]*TopicDetail,
+	timeout time.Duration,
+	validateOnly bool,
```

Comment:
😬 this is a breaking change.

Fortunately, it seems no one is actually using this exported function though.